featuretools/entityset/entityset.py

```diff
@@ -1008,66 +1006,79 @@ def add_last_time_indexes(self, updated_dataframes=None):
     # #  Other ###############################################
     # ###########################################################################
 
-    # def add_interesting_values(self, max_values=5, verbose=False, entity_id=None):
-    #     """Find interesting values for categorical variables, to be used to generate "where" clauses
+    def add_interesting_values(self, max_values=5, verbose=False, dataframe_id=None, values=None):
+        """Find interesting values for categorical variables, to be used to generate "where" clauses
```

Comment:
Not sure if we want to say "categorical columns" here or something like "columns that are categorical in nature" or something more woodwork-specific.

Reply:
I think saying something like  "categorical columns" or "columns that are categorical in nature" is good. We want to avoid saying Woodwork-specific things in Featuretools to avoid putting a burden on the end-user to learn Woodwork.

Reply:
Updated to `categorical columns`